nuget/lib/dependabot/nuget/update_checker/repository_finder.rb

```diff
@@ -195,7 +195,7 @@ def handle_timeout(repo_metadata_url:)
       def credential_repositories
         @credential_repositories ||=
           credentials
-          .select { |cred| cred["type"] == "nuget_feed" }
+          .select { |cred| cred["type"] == "nuget_feed" && cred["url"] && cred["token"] }
```

Comment:
```suggestion
          .select { |cred| cred["type"] == "nuget_feed" && cred["url"] }
```

As per the discussion in the linked PR, the `token` field will probably not be present, but `url` is absolutely required.